lib/floe/workflow/runner/kubernetes.rb

```diff
@@ -102,6 +102,33 @@ def cleanup(runner_context)
           delete_secret(secret) if secret
         end
 
+        def wait(timeout: nil, events: %i[create update delete])
+          watcher = kubeclient.watch_pods(:namespace => namespace)
+
+          timeout_thread = Thread.new { sleep(timeout); watcher.finish } if timeout.to_i > 0
```

Comment:
rubocop doesn't like this, and while I like conciseness perhaps we should expand it out (especially if we need more lines later)

```suggestion
          if timeout.to_i > 0
            timeout_thread = Thread.new do
              sleep(timeout)
              watcher.finish
            end
          end
```